### DISCUSSION TITLE
Blue Cats Patchwork

### DISCUSSION THREAD

**Discussion by emerymastering** [apohl79/audiogridder#850] [Help]
Hello Sir,

In Blue Cats patchwork all plugins are displayed very wrong. Attaching the printscreen. Any ideas?
![patch](https://user-images.githubusercontent.com/66089231/145781215-72978787-f0b8-4d8f-98eb-3b60e584e668.png)
.
Cheers
Denis

**Comment by Henri75** [comment]
I had the same issue with acustica nebula plugins.
Wrong position for screen capture.
Did you try some simple plugins from others like softube ?

**Comment by emerymastering** [comment]
It is same with any other plugin In Patchwork using AGridder.

Cheers
Denis "Emery" Teluchin
Director / Head Engineer
Mastering.lt
